openapi/ash-api.yaml

```diff
@@ -0,0 +1,19 @@
+openapi: "3.0.0"
+
+info:
+  title: "Ash API"
+  version: "0.0.1"
+
+servers:
+  - url: "/api/v0"
+
+paths:
+  "/info":
+    get:
+      operationId: "thoth.ash.api.info_get"
```

Comment:
This will work better if an automatically generated client will be used:

```
      x-openapi-router-controller: thoth.ash.api
      operationId: info_get
```

Reply:
Moved to automatic routing with Resty Resolver in new commit.  https://connexion.readthedocs.io/en/latest/routing.html#automatic-routing